### PR TITLE
add FAQ regarding X-Forwarded-Port when using OpenID

### PR DESCRIPTION
**Pull request by jmazzitelli**
see: https://github.com/kiali/kiali/issues/3111

### CODE REVIEW

**Comment by jmazzitelli** [comment]
This is what was added, so you will want to review this: https://deploy-preview-291--kiali.netlify.app/documentation/staging/faq/#x-forwarded-port

File: content/documentation/staging/faq/authentication/x-forwarded-port.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: How to forward OpenID login to Kiali behind a Virtual Service
```
Comment (israel-hdez) [inline content/documentation/staging/faq/authentication/x-forwarded-port.adoc:2]: I think we want a more generic focus, as this is not specific to VirtualServices. Something like:

```suggestion
title: How to configure the originating port when Kiali is served behind a proxy (OpenID support)
```

but rephrase as you see better.

File: content/documentation/staging/faq/authentication/x-forwarded-port.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+weight: 2
+---
+:icons: font
+
+When deploying Kiali behind an Istio VirtualService, the OpenID login button redirects to `<schema>://<web_fqdn>:20001`. If your ingress is listening on a different port, you can indicate this port via the `X-Forwarded-Port` HTTP header.
```
Comment (israel-hdez) [inline content/documentation/staging/faq/authentication/x-forwarded-port.adoc:7]:
```suggestion
When using OpenID strategy for authentication and deploying Kiali behind a reverse proxy or a load balancer, Kiali needs to know the originating port of client requests. You may need to setup your proxy to inject a `X-Forwarded-Port` HTTP header when forwarding the request to Kiali.
```

File: content/documentation/staging/faq/authentication/x-forwarded-port.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+For example, the VirtualService should have something like this to indicate the user should be forwarded to port 443 instead of 20001:
```
Comment (israel-hdez) [inline content/documentation/staging/faq/authentication/x-forwarded-port.adoc:9]:
```suggestion
For example, when using an Istio Gateway and VirtualService to expose Kiali, you could use the [headers property](https://istio.io/latest/docs/reference/config/networking/virtual-service/#Headers) of the route:
```

**Comment by jmazzitelli** [comment]
I committed the suggestions - had to make a minor fix  (the link syntax needed to be in adoc format). I'll merge once I see netlify's final preview and it looks good.

**Pull request merged**